BitwardenShared/UI/Vault/Vault/VaultList/VaultListItem.swift

```diff
@@ -143,7 +143,24 @@ extension VaultListItem {
             return ""
         }
     }
-
+    
+    var vaultItemAccessibilityId: String {
+        switch itemType {
+        case let .group(vaultListGroup, _):
+            if vaultListGroup.isFolder {
+                return "FolderCell"
+            } else if vaultListGroup.collectionId != nil {
+                return "CollectionCell"
+            } else {
+                return "ItemFilterCell"
+            }
+        case .cipher:
+            return "CipherCell"
+        case .totp:
+            return "TOTPCell"
+        }
+    }
```

Comment:
⛏️ Add docs.
⛏️ Add unit tests.
🎨 When using `return` in `if`s then you don't need the `else` keyword as the previous part would have returned if it was true. So you can write directly:
```swift
if vaultListGroup.isFolder {
    return "FolderCell"
} 
if vaultListGroup.collectionId != nil {
    return "CollectionCell"
}
return "ItemFilterCell"
```

Reply:
Done!